modelexpress_client/python/modelexpress/metadata_client.py

```
@@ -0,0 +1,464 @@
+            model_name.lower()
+            .replace("/", "-")
+            .replace("_", "-")
+            .strip("-")
+        )
+    
+    def publish_ready_signal(
+        self,
+        model_name: str,
+        worker_id: int,
+        session_id: str,
+        metadata_hash: str | None = None,
```

Ruff (0.15.0)

[warning] Unused method argument: `metadata_hash`

(ARG002)